src/frontend/src/lib/components/receive/ReceiveButtonWithModal.svelte

```diff
@@ -0,0 +1,16 @@
+<script lang="ts">
+	import { modalStore } from '$lib/stores/modal.store';
+	import ReceiveButton from '$lib/components/receive/ReceiveButton.svelte';
+
+	export let openModal: (modalId: symbol) => void | Promise<void>;
```

Comment:
Few naming ideas which feel like it could make things less verbose but, just sharing in case you like it.

```
export let openModal => export let open
export let specificModalId => export let modalId

const modalId => const definedModalId (or anything else)
```

Reply:
yes please! i even asked chatGPT for better names but nothing good

I will use your suggestions